docs/source/load-balancing/load-balancing.md

```diff
@@ -15,11 +15,13 @@ using `SessionBuilder::host_filter` method.
 ## Plan
 
 When a query is prepared to be sent to the database, the load balancing policy
-constructs a load balancing plan. This plan is essentially a list of nodes to
+constructs a load balancing plan. This plan is essentially a list of pairs of node and optional shard to
```

Comment:
Nit:
1) please keep maximum line width consistent
2) I suggest: *a list of pairs <node, optional shard>*

Reply:
Done